velox/common/dynamic_registry/tests/CMakeLists.txt

```diff
@@ -24,40 +24,69 @@ add_library(velox_overload_int_function_dynamic SHARED DynamicIntFunctionOverloa
 add_library(velox_overload_varchar_function_dynamic SHARED DynamicVarcharFunctionOverload.cpp)
 add_library(velox_function_non_default_dynamic SHARED DynamicFunctionNonDefault.cpp)
 
-set(CMAKE_DYLIB_TEST_LINK_LIBRARIES fmt::fmt Folly::folly glog::glog xsimd)
+set(DYLIB_TEST_LINK_LIBRARIES fmt::fmt glog::glog xsimd)
```

Comment:
Removing the `CMAKE_` prefix because this is not a CMake provided variable.
This is not a required change. This just improves readability.

Reply:
Shouldn't this just be an interface target?